json/src/main/scala/io/sphere/json/ToJSON.scala

```diff
@@ -88,10 +87,29 @@ object ToJSON {
   }
 
   implicit val moneyWriter: ToJSON[Money] = new ToJSON[Money] {
-    def write(m: Money): JValue = JObject(
-      JField("currencyCode", toJValue(m.currency)) ::
-      JField("centAmount", toJValue(m.centAmount)) :: Nil
-    )
+    def write(m: Money): JValue = JObject(List(
+      JField("type", toJValue(m.`type`)),
+      JField("currencyCode", toJValue(m.currency)),
+      JField("centAmount", toJValue(m.centAmount)),
+      JField("fractionDigits", toJValue(m.currency.getDefaultFractionDigits))
```

Comment:
I don't find this `fractionDigits` documented anywhere in our previous meetings. Where does it come from?
Why do we need that for the current `Money`?

Reply:
You are right, we haven't discussed it explicitly yet. As i was implementing it, i thought that it might be a good idea to add `fractionDigits` in the `Money` type as well for the sake of API consistency and client convenience (so that client does not need to figure out the amount of fraction digits based on the currency itself). 

Do you think it is a good idea? I don't have a strong opinion about this one, so I can remove it if you think that it would be better to avoid it.